RevenueCatUI/Templates/V2/Components/Root/RootView.swift

```diff
@@ -59,6 +68,11 @@ struct RootView: View {
             self.sheetViewModel = sheet
         })
         .bottomSheet(sheet: $sheetViewModel, safeAreaInsets: self.safeAreaInsets)
+        .onChangeOf(sheetViewModel) { newValue in
+            if newValue == nil {
+                packageContext.package = defaultPackage
+            }
+        }
```

Comment:
I think this deserves a comment explaining why `if newValue == nil {` is needed, or when is set back to nil ?